apps/src/templates/certificates/certificate_batch.module.scss

```diff
@@ -1,5 +1,6 @@
 @import '2022-rebrand-update';
-@import 'color', 'font', 'typography', 'breakpoints';
+@import 'color', 'typography', 'breakpoints';
```

Comment:
`shared/css/typography.scss` is a copy of `frontend/packages/component-library-styles/typography.module.scss` added to allow to use it in pegasus previously, we might be able to even remove it once we figure out how to use dsco styles in pegasus now. 
Meaning we should be good to use `frontend/packages/component-library-styles/typography.module.scss` here